Formula/gitmoji.rb

```diff
@@ -0,0 +1,18 @@
+require "language/node"
+
+class Gitmoji < Formula
+  desc "Emoji guide for your commit messages"
+  homepage "https://gitmoji.carloscuesta.me"
+  url "https://registry.npmjs.org/gitmoji-cli/-/gitmoji-cli-1.9.2.tgz"
+  sha256 "af7a6c7b0874d2971be909c9c2166b94db4fe1c3d1500f49cfac30d0a5d1b433"
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_equal "Gitmojis updated successfully!", shell_output("#{bin}/gitmoji -u").chomp
```

Comment:
Homebrew does not include software that autoupdates, as it messes with our own versioning/update system. The software has to be installed in a way that disables its autoupdate capabilities.

Reply:
Hi @fxcoudert, Thank you for your feedback. 

Running `gitmoji --update` does not mean updating the software itself but updating the emoji lists cached locally (for offline usage).
A better test should have been to run `gitmoji --list` and check for a particular result but I was a bit worried about stability of this test that would depend on the emoji list expected to evolve.

<img width="535" alt="screenshot 2018-11-03 at 11 28 01" src="https://user-images.githubusercontent.com/2805359/47951043-9800b000-df5b-11e8-99aa-11405f9df08d.png">

If you have a better way to test in mind, please let me in

Reply:
I suggest checking for `🐛 - :bug: - `, which is not likely to change.

Reply:
Ok, done in https://github.com/Homebrew/homebrew-core/pull/33660/commits/4f7455a6fa56fec21082aa6e19e7170efbe6d376